exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/requirements_dependencies.txt

```diff
@@ -8,4 +8,4 @@ jupysql==0.10.7
 sqlalchemy_exasol==4.6.3
 stopwatch.py==2.0.1
 --extra-index-url https://download.pytorch.org/whl/cpu
-git+https://github.com/exasol/notebook-connector.git@0.2.6
+git+https://github.com/exasol/notebook-connector.git@aa1496f
```

Comment:
Update to a released version if possible

Reply:
We have unfortunately a small issue with the release notebook connector which can't be resolved in time. It is still open how exactly we handle this. Probably, we will simply create a tag without a Github Release until, we can resolve the root issue.